src/main/java/com/openlattice/shuttle/util/Constants.java

```diff
@@ -37,5 +37,10 @@
     public static final String TYPE                = "type";
     public static final String UPDATE_TYPE         = "updateType";
     public static final String VALUE               = "value";
+    public static final String LEFTCOLUMN          = "leftColumn";
```

Comment:
Our convention is to use `_` between words, so `LEFT_COLUMN` instead of `LEFTCOLUMN`. Please update these constants.